src/wiki.rs

```diff
@@ -210,7 +215,23 @@ impl Wiki {
     }
 
     pub fn read_file(&self, file_path: &[&str]) -> Result<Vec<u8>, MyError> {
-        self.0.repository.read_file(file_path)
+        lazy_static! {
+            static ref RE: Regex = Regex::new(r"\[\[(.+?)\]\]").unwrap();
```

Comment:
```suggestion
            static ref RE: Regex = Regex::new(r"^{{([^}]+?)}}$").unwrap();
```

Basically this is what I'm asking for in my long comment.

Reply:
I think the regex should be `^{{(.+?)}}$`, not `^{{([^}]+?)}}$`.

Mine matches `{{a}b}}` which could possibly be a valid filename, albeit a weird one.

I can use yours if you prefer it.

Reply:
Good point.

Reply:
I note you wanted `{{filename.md}}` as the transclusion.  Should the regex be: `^{{(.?+\.md)}}$` instead?  This will reduce (but not eliminate) false positives.

> I do see other syntaxes:

I feel like the square braces are a little more in keeping with Markdown vs the curly braces of Mediawiki but I don't feel like it makes that much difference.

Reply:
I think it's fine to allow any extension. For example, this could be handy for including a code file into a code block like this:

      ```
      {{script.sh}}
      ```